src/main/kotlin/no/uio/microobject/data/TripleManager.kt

```diff
@@ -146,6 +157,21 @@ class TripleManager(private val settings: Settings, val staticTable: StaticTable
         return model
     }
 
+    private fun getTripleStoreOntologyAsModel(): Model {
+        // Test case only
+        if (testModel != null) return testModel as Model
+        // Normal behaviour with a Fuseki environment
+        return RDFConnectionFactory.connect(settings.tripleStore + "/data").fetch()
+    }
+
+    /**
+     * Regenerate the triple store model. We'll do so by fetching again the data
+     * This will be called when the triple store is updated, and we want to update the model.
+     */
+    fun regenerateTripleStoreModel(): Unit {
```

Comment:
Please add a statement to the language "reload" that updates the internal model so we can do this from inside a program